test/e2e/configuration_anomaly_detection_test.go

```diff
@@ -379,4 +380,82 @@ var _ = Describe("Configuration Anomaly Detection", Ordered, func() {
 		Expect(lsResponseAfter.Items().Len()).To(BeNumerically(">", lsReasonsBefore),
 			"Expected more limited support reasons after infrastructure node shutdown")
 	})
+
+	It("AWS CCS: clustermonitoringerrorbudgetburn (misconfig + revert scenario)", func(ctx context.Context) {
+		if provider == "aws" {
+			const (
+				namespace     = "openshift-user-workload-monitoring"
+				configMapName = "user-workload-monitoring-config"
+			)
+
+			// STEP 0: Get cluster information from OCM
+			fmt.Println("Step 0: Fetching cluster info")
+			response, err := ocme2eCli.ClustersMgmt().V1().Clusters().Cluster(clusterID).Get().Send()
+			Expect(err).ToNot(HaveOccurred(), "Failed to get cluster from OCM")
+			cluster := response.Body()
+			Expect(cluster).ToNot(BeNil(), "Cluster response is nil")
+
+			// STEP 1: Get service logs BEFORE misconfiguration
+			fmt.Println("Step 1: Getting service logs before misconfiguration")
+			logs, err := utils.GetServiceLogs(ocmCli, cluster)
+			Expect(err).ToNot(HaveOccurred(), "Failed to fetch service logs before misconfig")
+			logsBefore := logs.Items().Slice()
+
+			// STEP 2: Fetch and backup the current ConfigMap
+			fmt.Println("Step 2: Backing up current ConfigMap")
+			originalCM := &corev1.ConfigMap{}
+			err = k8s.Get(ctx, configMapName, namespace, originalCM)
```

Comment:
```suggestion
			err = k8s.Get(ctx, configMapName, namespace, originalCM)
```

The utils file got a FetchConfigMap function - it should either be used or removed from the utils.go